_grab_ui.py

```diff
@@ -690,7 +711,7 @@ def ospec() -> str:
         loop = Loop()
         loop.loop(handler)
         if loop.return_code == 0 and 'copy' in handler.result:
-            sys.stdout.buffer.write(b''.join((b'\x1b]52;c;',
+            sys.stdout.buffer.write(b''.join((b'\x1b]52;' + handler.copy_to + b';',
                                               b64encode(handler.result['copy'].encode('utf-8')),
                                               b'\x1b\\')))
```

Comment:
Conceptually, there is already concatenation going on here, and it’s expressed as a `join`. No need to use both `join` and `+` in the same expression.